tools/.golangci.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 run:
   timeout: 30m
-  skip-files: [^zz_generated.*]
+  issues.exclude-files: [^zz_generated.*]
```

Comment:
should this be put under issues section?

Reply:
Fixed according to the provided link :) 

I kept `^zz_generated.*` instead of `/zz_generated_` from the provided link though.